lib/bundle-manager.js

```diff
@@ -284,16 +284,22 @@ function _handleChange(bundleName) {
 		log.debug('Processing change event for', bundleName);
 		resetBackoffTimer();
 
-		let reparsedBundle;
-		const bundleCfgPath = path.join(_cfgPath, `${bundleName}.json`);
-		if (fs.existsSync(bundleCfgPath)) {
-			reparsedBundle = parseBundle(bundle.dir, bundleCfgPath);
-		} else {
-			reparsedBundle = parseBundle(bundle.dir);
-		}
+		try {
+			let reparsedBundle;
+			const bundleCfgPath = path.join(_cfgPath, `${bundleName}.json`);
+
+			if (fs.existsSync(bundleCfgPath)) {
+				reparsedBundle = parseBundle(bundle.dir, bundleCfgPath);
+			} else {
+				reparsedBundle = parseBundle(bundle.dir);
+			}
 
-		module.exports.add(reparsedBundle);
-		emitter.emit('bundleChanged', reparsedBundle);
+			module.exports.add(reparsedBundle);
+			emitter.emit('bundleChanged', reparsedBundle);
+		} catch (error) {
+			log.warn('Unable to handle the bundle "%s" change: %s', bundleName, error.message, {error});
```

Comment:
Any particular reason that `error` is passed as a property of a new object, instead of directly?

Reply:
That's just something I do when I'm logging messages, especially when I want to associate multiple properties.

Reply:
We can leave it for now. Who knows, I might end up liking it better than my normal way of logging things.